### ISSUE TITLE
twemoji.twitter.com no longer acceccible

### ISSUE THREAD

**Issue by agscs** [twitter/twemoji#641]
**Describe the bug**
https://twemoji.twitter.com is no longer acceccible

**To Reproduce**
1. Go to [https://twemoji.twitter.com](https://twemoji.twitter.com)
2. See error

**Screenshots**
![image](https://github.com/twitter/twemoji/assets/1184350/e133ff4e-1ee8-48ee-8997-a1532983690d)

**Comment by ThisCatLikesCrypto** [comment]
I'm pretty sure the page is just plain gone because otherwise https://twemoji.x.com would work and it doesn't

**Comment by agscs** [comment]
yeah i did try both, but its looks like they didnt re-add it back

**Comment by CanOfDew132** [comment]
another screenshot:
<img width="960" alt="Screenshot 2024-06-14 233005" src="https://github.com/twitter/twemoji/assets/165729285/01ebeb55-aada-4652-bed3-bf2d987f8012">

**Comment by agscs** [comment]
Found this but its old
[https://twitter.github.io/twemoji/](https://twitter.github.io/twemoji/)

**Comment by CanOfDew132** [comment]
> Found this but its old
https://twitter.github.io/twemoji/

thanks :)

+ image
<img width="960" alt="Screenshot 2024-06-17 211746" src="https://github.com/twitter/twemoji/assets/165729285/1af24503-1590-462d-920a-bdc1278979f5">

**Comment by AlexAnimateMp4** [comment]
For the twemoji.js file or its derivatives, knowing that twemoji.maxcdn.com is down, the value of the twemoji.base variable can be temporarily replaced by https://raw.githubusercontent.com/twitter/twemoji/master/assets/

**Comment by shuuji3** [comment]
Here's the archive page by Internet Archive: Twemoji - https://web.archive.org/web/20220214094446/https://twemoji.twitter.com/

I suspect it's unlikely the website will be recovered since the official Twitter account for OSS was also deleted https://twitter.com/TwitterOSS and X Corp. seems to have abandoned this open-source development.

However, the latest Twemoji fork is still actively maintained and updated by former Twitter designers and Discord/WordPress designers on the fork repository here: https://github.com/jdecked/twemoji

They offer the GitHub pages and the updated (it has a new emoji copy feature) preview gallery too: 
- Twitter Emoji (Twemoji) - https://jdecked.github.io/twemoji/
- Twitter Emoji (Twemoji) Preview - https://jdecked.github.io/twemoji/v/latest/preview-svg.html

For reference, this article explains the recent development of Twemoji:

- X Redesigns Water Pistol Emoji Back To A Firearm - https://blog.emojipedia.org/x-redesigns-water-pistol-emoji-back-to-a-firearm/

**Comment by Monoverde888** [comment]
duplicate of #634 🙄
